js/components/addressBookList/addressBookList_controller.js

```diff
@@ -7,6 +7,14 @@ angular.module('contactsApp')
 	AddressBookService.getAll().then(function(addressBooks) {
 		ctrl.addressBooks = addressBooks;
 		ctrl.loading = false;
+		if(ctrl.addressBooks.length === 0) {
+			AddressBookService.create(t('contacts', 'Contacts')).then(function() {
```

Comment:
I assume you should return a promise here so that the client code is waiting until this new address book is ready.